rds-monitoring/templates/prometheusrules.yaml

```diff
@@ -94,3 +94,12 @@ spec:
           annotations:
             message: 'Replica lag for RDS instance {{`{{ $labels.dbinstance_identifier }}`}} is higher than 30 seconds.'
             runbook_url: 'https://github.com/skyscrapers/documentation/tree/master/runbook.md#alert-name-rdsreplicalaghigh'
+        - alert: RDSBurstBalanceLow
+          expr: aws_rds_burstbalance_average{job="{{ .Release.Name }}-prometheus-cloudwatch-exporter"} {{ include "rds-monitoring.filterOnTag" . }} < 20
+          for: 10m
+          labels:
+{{ include "rds-monitoring.alertLabels" . | indent 12 }}
+            severity: info
```

Comment:
Hmmm shouldn't this be higher severity? Seems quite important to me no?

Reply:
what do you propose? Warning? I wouldn't do critical for this.

Reply:
idd, probably not critical... :thinking: 
then let's go for warning

Reply:
done